examples/scripts/quickstart.py

```diff
@@ -143,16 +140,31 @@ def make_predictions(client, predictor_host, queries):
 
     print('Adding models to Rafiki...') 
     create_model(client, 'TfFeedForward', task, 'examples/models/image_classification/TfFeedForward.py', \
-                'TfFeedForward', dependencies={ ModelDependency.TENSORFLOW: '1.12.0' })
+                'TfFeedForward', dependencies={ ModelDependency.TENSORFLOW: '1.12.0' }, \
+                access_right=ModelAccessRight.PUBLIC)
     create_model(client, 'SkDt', task, 'examples/models/image_classification/SkDt.py', \
-                'SkDt', dependencies={ ModelDependency.SCIKIT_LEARN: '0.20.0' })
-
-    print('Logging in as app developer...')
-    client.login(email=APP_DEVELOPER_EMAIL, password=USER_PASSWORD)
+                'SkDt', dependencies={ ModelDependency.SCIKIT_LEARN: '0.20.0' }, \
+                access_right=ModelAccessRight.PRIVATE)
 
     print('Creating train job for app "{}" on Rafiki...'.format(app)) 
+    models = [
```

Comment:
@nudles any foreseeable issues with moving budget specification to be per-model?

Reply:
I don't have such scenarios right now.
if we want to train a specific model for 4 trials, we can set the global budget and configure the model.
so, better keep it as a global configuration.

Reply:
@nudles  @nginyc 

Regarding the global budget, do i interpret it as each model has the global budget each, or global budget for all models?

Reply:
it should be one budget per job.
if the budget is 4 trials, then we train each model for 4 trials or in total 4 trials for all models.
@nginyc which is the current one?

Reply:
For the 4 trials for all models, I can foresee a problem when the graph is integrated and there is not enough trials to cover the entire graph.

Reply:
the current logic is e.g. if `budget` = `{ BudgetType.MODEL_TRIAL_COUNT: 2 }`, each model does 2 trials

Reply:
Currently, i fixed it such that there is a global budget, and also allows overriding of budget for individual models. @nudles @nginyc Is this fine?

Reply:
ok.